outboxrepo/migrations/000001_create_foundation_outbox_events.up.sql

```diff
@@ -1,8 +1,10 @@
 CREATE TABLE foundation_outbox_events (
-    id BIGINT PRIMARY KEY,
+    id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
     topic TEXT NOT NULL,
     key TEXT NOT NULL,
+    proto_name TEXT NOT NULL,
     payload BYTEA NOT NULL,
     headers JSONB NOT NULL,
+    correlation_id TEXT NOT NULL,
```

Comment:
Correlation is, as well, already in headers (or should be, if it's not), I can't recall right now if I implemented this, so it's better to check.

Reply:
Done, now using headers